js/app.js

```diff
@@ -54,12 +56,24 @@ function fromTaggedBase64(tb64) {
     const data = bytes.subarray(0, n - 1);
     const cs2 = crc8('CRC-8', bytesConcat(toBytes(tag), data)) ^ (data.length % 256);
     if (cs == cs2) {
-        return [tag, stringFromBytes(data)];
+        return [tag, data];
     } else {
         return undefined;
     }
 }
 
+/// Given a TaggedBase64 string with a value encoding a UTF-8 string,
+/// parse and extract the tag and the UTF-8 string. Otherwise,
+/// return undefined.
```

Comment:
This doesn't return `undefined` if the value isn't actually a UTF-8 string. I don't think that's important for the purposes of this example, though, if it's just meant to show how you would convert from UTF-8

Reply:
I'll fix this when adding the tests from [issue 23](https://github.com/EspressoSystems/tagged-base64/issues/23)